Remotion/Validation/Core/Results/PropertyValidationFailure.obsolete.cs

```diff
@@ -17,30 +17,34 @@
 using System;
 using JetBrains.Annotations;
 using Remotion.Reflection;
-using Remotion.Utilities;
 
 namespace Remotion.Validation.Results
 {
+  [Obsolete("Use ValidationFailure instead. (Version 5.0.0)", true)]
```

Comment:
I added obsolete stub implementations for the removed types. That's something we do for important public APIs to allow easier upgrade because you get the information about what do directly in the compiler error.